src/windows/main_window.py

```diff
@@ -889,8 +896,21 @@ def actionSaveFrame_trigger(self, event):
             recommended_path = info.HOME_PATH
 
         #log.info("Saving frame to %s/%s.png" % (recommended_path, self.preview_thread.current_frame))
-        framePath = "%s/%s.png" % (recommended_path, self.preview_thread.current_frame)
-        log.info("Saving frame to %s" % framePath )
+        framePath = _("%s/Frame%04d.png" % (recommended_path, self.preview_thread.current_frame))
```

Comment:
I wonder if this should use, and update, the project `export_path`, instead of always starting at `recommended path`? Or perhaps even add yet another project file variable, to track `snapshot_path` or whatever? 

(See #1372 which created `import_path` to track last-open-location for "Import Files...". But `export_path` already exists, so you could just use that. Assuming users won't be annoyed that their exports and their frame snapshots affect the other's starting location.)

If nothing else, subsequent invocations of the dialog within the same session should endeavor to start in the same directory where the previous file was saved. Making the user navigate there from `recommended_path` every single time can seem cruel, especially if they have to traverse a long path to the intended destination. Doing that _once_ is tedious. Having to do it more than once feels like abuse.

Whether that persists beyond the current session... up to you, I don't really have an opinion on that.